debian/changelog

```diff
@@ -1,3 +1,96 @@
+flatpak (0.11.3-3endless0) unstable; urgency=medium
+  * New upstream releases
```

Comment:
An alternative approach here would be to:
- reset the whole changelog to debian 0.11.3
- add on top our own changelog entry with a description of the changes that we still carry on top of debian packaging

I find that easier when rebasing and having to consider whether changes are still required going forward, but Iguess it's ultimately up to your preference.

Reply:
I think having the whole history makes sense.